executors_test.go

```diff
@@ -533,6 +534,21 @@ func Test_Create_Non_PK_ID(t *testing.T) {
 	})
 }
 
+func Test_Create_Parallel(t *testing.T) {
```

Comment:
As you can see, this test failed first with the data race: https://github.com/gobuffalo/pop/runs/7304729964?check_suite_focus=true#step:6:47